sdk/containerregistry/container-registry/test/utils/utils.ts

```diff
@@ -50,6 +50,28 @@ export const recorderEnvSetup: RecorderEnvironmentSetup = {
   ]
 };
 
+function wlog(message: string, prefix: string = ""): void {
+  let s = [];
+  for (let i = 0; i < message.length; i++) {
+    s.push(message.charAt(i));
+  }
+  console.log("-#-", prefix, s.join(" "));
+}
+
+function getAuthority(endpoint: string): AzureAuthorityHosts {
```

Comment:
Will this work if the endpoint uses a custom domain?

Reply:
No, but do we need to test custom domain in weekly pipeline?

Reply:
Ah, apologies, I didn't realize this was just for the test pipeline! :)